mitodl/micromasters
# changing micromasters certificate logo in program certificate

#### Pre-Flight checklist


#### What are the relevant tickets?
https://github.com/mitodl/micromasters/issues/4842

#### What's this PR do?
Changes logo on micromasters program certificates with some minor changes in design.

#### How should this be manually tested?
Create a program certificate and see if the new logo has been replaced perfectly with the old one with the desired design change in the ticket https://github.com/mitodl/micromasters/issues/4842

#### Screenshots (if appropriate)
Program Certificate Before:
![image](https://user-images.githubusercontent.com/42243411/112464440-528bea00-8d85-11eb-9730-e3583ddf612d.png)

Program Certificate After:
![image](https://user-images.githubusercontent.com/42243411/112464302-24a6a580-8d85-11eb-908b-4ee6b760661c.png)

Program Certificate Before (Printed):
![image](https://user-images.githubusercontent.com/42243411/112464422-4e5fcc80-8d85-11eb-9120-5d4e696e132e.png)

Program Certificate After (Printed):
![image](https://user-images.githubusercontent.com/42243411/112464261-16588980-8d85-11eb-83c4-d7ceb73f719f.png)
